crates/storage/db/src/tables/models/blocks.rs

```diff
@@ -40,10 +40,20 @@ impl StoredBlockBody {
         self.start_tx_id.saturating_add(self.tx_count).saturating_sub(1)
     }
 
+    /// First transaction index.
+    pub fn first_tx_index(&self) -> TxNumber {
```

Comment:
Where is this and `tx_count` used? I would be careful about using them since we have footgunned ourselves multiple times by manually constructing ranges, which is why we added the other functions in here.

Reply:
Used for reading transaction of multiple blocks:
https://github.com/paradigmxyz/reth/blob/062cbf8a50b7e90191899c825ad7988a194d6c09/crates/storage/provider/src/transaction.rs#L545-L550

I am using `..=` so it is correct even if second item is less the first: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=952d632514ed124d410a4e432871b4f4